app/src/main/java/de/westnordost/streetcomplete/quests/barrier_bicycle_barrier_installation/AddBicycleBarrierInstallation.kt

```diff
@@ -0,0 +1,37 @@
+package de.westnordost.streetcomplete.quests.barrier_bicycle_barrier_installation
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.filter
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement.BICYCLIST
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement.BLIND
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement.WHEELCHAIR
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement.LIFESAVER
+import de.westnordost.streetcomplete.osm.Tags
+
+class AddBicycleBarrierInstallation : OsmFilterQuestType<BicycleBarrierInstallationAnswer>() {
+
+    override val elementFilter = "nodes, ways with barrier = cycle_barrier and cycle_barrier and cycle_barrier != tilted and !cycle_barrier:installation"
+    override val changesetComment = "Specify cycle barrier installation"
+    override val wikiLink = "Key:cycle_barrier:installation"
+    override val icon = R.drawable.ic_quest_no_bicycles
+    override val isDeleteElementEnabled = true
+
+    override val achievements = listOf(BLIND, WHEELCHAIR, BICYCLIST, LIFESAVER)
```

Comment:
Please remove BLIND and WHEELCHAIR

Reply:
OK, if you wish. Note: those were copied from somewhat similar `AddBarrierType.kt`, and made sense to me - e.g. `cycling barrier` will stop wheelchair user as efficiently (or more so) as `bollard` will...

Reply:
removed in 1f97b2b2dd024aa2d82315a3324364998494a19a